new_project/examples/separate_apps/src/taker.ts

```diff
@@ -0,0 +1,96 @@
+import { SwapRequest } from "comit-sdk";
+import { formatEther } from "ethers/utils";
+import moment from "moment";
+import readLineSync from "readline-sync";
+import { toBitcoin, toSatoshi } from "satoshi-bitcoin-ts";
+import { Actor, checkEnvFile, startClient } from "./lib";
+
+(async function main() {
+    checkEnvFile(process.env.DOTENV_CONFIG_PATH!);
+
+    const taker = await startClient(1);
+
+    console.log(
+        "[Taker] Bitcoin balance: %f. Ether balance: %f",
+        parseFloat(await taker.bitcoinWallet.getBalance()).toFixed(2),
+        parseFloat(
+            formatEther(await taker.ethereumWallet.getBalance())
+        ).toFixed(2)
+    );
+
+    const peerId = readLineSync.question("What is the Maker's Peer Id?");
```

Comment:
❓Not sure, but did we talk about if we can automate this completely? 
If so, do we want to have a complete automated example here?

Reply:
I think the result of the discussion is that we'll leave this discussion out of the scope of this PR.

Reply:
Yap, sounds good.